hypervisor/src/mshv/mod.rs

```diff
@@ -1576,6 +1577,11 @@ impl cpu::Vcpu for MshvVcpu {
 
         Ok(())
     }
+
+    #[cfg(feature = "kvm")]
+    unsafe fn get_kvm_vcpu_raw_fd(&self) -> RawFd {
+        todo!()
+    }
```

Comment:
perhaps `unreachable!` or something like that? I am guessing that `mshv` should not be used when the `kvm` feature is set?